main.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/cookiejar"
+	"os"
+	"time"
+
+	"github.com/xuri/excelize/v2"
+)
+
+const (
+	thirtyDays  int = 30
+	rowOffset   int = 1
+	sheetOffset int = 1
+)
+
+type Input struct {
+	Filename       string
+	SheetName      string
+	UsernameHeader string
+	PasswordHeader string
+	SheetPassword  string
+}
+
+type Portal struct {
+	*Input
+	Environment string
+	Hostname    string
+	IdmHostname string // identity management hostname
+	infoLog     *log.Logger
+	errorLog    *log.Logger
+}
+
+type Creds struct {
+	OldPassword string
+	NewPassword string
+}
+
+var client *http.Client
```

Comment:
I don't think this needs to be a package variable. It is only used once in `main` so you can just create an initialize the client there (or in `resetPasswords` if you want)

Reply:
Added a function to create the client. Invoke the function in resetPasswords.